### ISSUE TITLE
"JSON encoding API usage" inspection works wrong when using PHP 8 named arguments feature

### ISSUE THREAD

**Issue by Stalinko** [kalessil/phpinspectionsea#1845]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Extended) v.4.0.7.1 |
| Language level | PHP 8.0+                   |

Current behaviour
--

```php
$decoded = json_decode('{"this is": "json"}', true, flags: JSON_THROW_ON_ERROR);
```

Shows a warning `Please consider taking advantage of JSON_THROW_ON_ERROR flag for this call options.`
If I click this warning then it will turn the code into:

```php
$decoded = json_decode('{"this is": "json"}', true, JSON_THROW_ON_ERROR, JSON_THROW_ON_ERROR);
```

Expected behaviour
--

It shouldn't show a warning because this argument is already set.

Environment details
---

PHP 8 Named Arguments feature: https://www.php.net/manual/en/functions.arguments.php#functions.named-arguments

**Comment by jdreesen** [comment]
Duplicate of #1725, #1639 and #1807
